### PR TITLE
Add styling for printing

### LINKED ISSUE #2988: Datamodel viewer not printing properly
Printing preview shows that tables are cut off.

image

https://coldfish-edge.test.specifysystems.org/specify/data-model/
branch: `edge`

### PR DESCRIPTION
**Pull request by CarolineDenis**
Fixed #2988

### CODE REVIEW

File: specifyweb/frontend/js_src/lib/localization/query.ts [hunk 1]
```diff
@@ -650,28 +650,8 @@ export const queryText = createDictionary({
   importNoReadPermission: {
     'en-us':
       'The query you imported contains tables you do not have read access to:',
-    'es-es': `
```
Comment (maxxxxxdlp) [inline specifyweb/frontend/js_src/lib/localization/query.ts:653]: Looks like the merge didn't go well once again. Not sure why this keeps on happening.
Do you want to revisit merge conflict resolution on Monday?

**Review by maxxxxxdlp — CHANGES_REQUESTED** [review]
Merge problems. Otherwise looks good

File: specifyweb/frontend/js_src/lib/localization/query.ts (hunk 1 above)
Comment (maxxxxxdlp) [reply specifyweb/frontend/js_src/lib/localization/query.ts]: Fixed by removing "virus" merge commits

**Review by maxxxxxdlp — APPROVED** [review]

**Review by grantfitzsimmons — APPROVED** [review]

**Comment by grantfitzsimmons** [comment]
Works in Firefox, Chrome, and Edge

**Pull request merged**